### DISCUSSION TITLE
Android 14上，偶尔关闭连接的时候会导致依赖webview的浏览器都提示net::ERR_PROXY_CONNECTION_FAILED

### DISCUSSION THREAD

**Discussion by 6543x1** [MetaCubeX/ClashMetaForAndroid#274] [General]
Android 14上，偶尔关闭连接的时候会导致依赖webview的浏览器都提示net::ERR_PROXY_CONNECTION_FAILED，必须重新开关代理才能解决。在Android10的设备上没有遇到此类问题，不知道是不是Android更新没有适配导致的。本人对Android开发不是很了解，不太清楚（

**Comment by ZLHuo** [comment]
同问

**Reply by 6543x1** [reply]
我改用flclash后就好了，可能是clash for android没适配android14导致的bug。clash meta for android只替换了core，所以也保留这个bug。

**Reply by supermansaga** [reply]
flclash哪里能下载？是这个吗？谢谢。Clash Meta for Android完全不能在Google TV Streamer w/ Android 14上安装，跟115一样. FlClash-0.8.68-android-arm64-v8a.apk似乎也不行 https://github.com/chen08209/FlClash

**Comment by Dystnine** [comment]
好像还是有这个问题
